src/signals/incident-management/containers/IncidentSplitContainer/IncidentSplitContainer.js

```diff
@@ -80,15 +82,15 @@ const IncidentSplitContainer = ({ FormComponent }) => {
       if (success) {
         dispatch(
           showGlobalNotification({
-            title: 'Deelmelding gemaakt',
+            title: 'Deelmelding(en) gemaakt',
             variant: VARIANT_SUCCESS,
             type: TYPE_LOCAL,
           })
         );
       } else {
         dispatch(
           showGlobalNotification({
-            title: 'De deelmelding kon niet gemaakt worden',
+            title: 'De deelmelding(en) kon niet gemaakt worden',
```

Comment:
Hmm, then it should be `kon(den)` as well...

Reply:
Yes.. which makes it less readable. I've discussed it with design and will change the text to the singular version (i.e. Deelmelding gemaakt). Even though in practice it can be multiple deelmeldingen, the message is clear and concise.